src/components/InputField/InputField.tsx

```diff
@@ -57,12 +84,13 @@ InputField.defaultProps = {
   label: "",
   fieldType: "TEXT",
   required: false,
+  formatting: false,
 };
 
 const VALIDATOR: { [key: string]: RegExp } = {
   POSTCODE: /^\d{5}$/,
-  AMOUNT: /^\d{1,7}$/,
-  PERCENTAGE: /^\d{1,3}%$/,
+  AMOUNT: /^\$?\d{1,7}$/,
```

Comment:
I'd avoid adding a $ checker in here, instead opt to make the dollar sign or currency symbol a default on all AMOUNT fields.

Reply:
Fixed.